CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## v0.0.3 - 2023-??-??
 
 * Enable support for root level NS records (`SUPPORTS_ROOT_NS=true`)
+* Enable support for wildcard zone lookups (list_zones())
```

Comment:
Minor tweak here, 0.0.3 is a work in progress so it can just go into this section until a release is cut.